src/schemas/gqlTypeDefs.ts

```diff
@@ -20,350 +20,350 @@
+		"""
+		clearClinicalRegistration(shortName: String!, registrationId: String!): Boolean!
+
+		"""
+		Complete registration of the currently uploaded Clinical Registration data
+		On Success, returns a list of the new sample IDs that were committed
+		"""
+		commitClinicalRegistration(shortName: String!, registrationId: String!): [String]!
+
+		"""
+		Clear Clinical Submission
+		fileType is optional, if it is not provided all fileTypes will be cleared. The values for fileType are the same as the file names from each template (ex. donor, specimen)
+		"""
+		clearClinicalSubmission(
+			programShortName: String!
+			version: String!
+			fileType: String
+		): ClinicalSubmissionData!
+
+		"""
+		Validate the uploaded clinical files
+		"""
+		validateClinicalSubmission(programShortName: String!, version: String!): ClinicalSubmissionData!
+
+		"""
+		- If there is update: makes a clinical submission ready for approval by a DCC member,
+		returning submission data with updated state
+		- If there is NO update: merges clinical data to system, returning an empty submission
+		"""
+		commitClinicalSubmission(programShortName: String!, version: String!): ClinicalSubmissionData!
+	}
+
+	scalar DateTime
+
+	"""
+	Query Variables for Pagination & Filtering
+	"""
+	input ClinicalInput {
+		page: Int!
+		pageSize: Int!
+		sort: String
+		entityTypes: [String]
+		donorIds: [Int]
+		submitterDonorIds: [String]
+		completionState: String
+	}
+
+	"""
+	Collated Clinical Data Query Response
+	"""
+	type ClinicalData {
+		programShortName: String!
+		clinicalEntities: [ClinicalDataEntities]!
+		completionStats: [CompletionStats]
+		clinicalErrors: [ClinicalErrors]!
+	}
+
+	"""
+	Clinical Data DonorId Search Query Response
+	"""
+	type ClinicalSearchData {
+		programShortName: String!
+		searchResults: [ClinicalSearchResults]!
+		totalResults: Int!
+	}
+
+	"""
+	Clinical Data DonorId Search Result Record
+	"""
+	type ClinicalSearchResults {
+		donorId: Int!
+		submitterDonorId: String
+	}
+
+	"""
+	Submitted Program Clinical Data arranged by Entity type
+	"""
+	type ClinicalDataEntities {
+		entityName: String!
+		totalDocs: Int!
+		records: [[ClinicalRecordField]]!
+		entityFields: [String]
+		completionStats: [CompletionStats]
+	}
+
+	"""
+	Data Submission / Schema Errors for a given Donor
+	"""
+	type ClinicalErrors {
+		donorId: Int
+		submitterDonorId: String
+		entityName: String
+		errors: [ClinicalErrorRecord]
+	}
+
+	"""
+	Specific Error Field + Values
+	"""
+	type ClinicalErrorRecord {
+		errorType: String
+		fieldName: String
+		index: Int
+		info: ClinicalErrorInfo
+		message: String
+		entityName: String
+	}
+
+	type ClinicalErrorInfo {
+		value: [String]
+		message: String
+	}
+
+	"""
+	Completion Data for a given Donor
+	"""
+	type CompletionStats {
+		coreCompletion: CoreCompletionFields
+		coreCompletionDate: String
+		coreCompletionPercentage: Float
+		hasMissingEntityException: Boolean
```

Comment:
This is the only change, updating the response structure based on the updated CompletionStats type.